### DISCUSSION TITLE
How to remove null properties on a vertex?

### DISCUSSION THREAD

**Discussion by smokedlinq** [Gremlinq/ExRam.Gremlinq#563] [Q&A]
In gremlin, I can do properties("propertyName").drop(), but is there a way to have Gremlinq do that automagically with Update()?

**Comment by danielcweber** [comment]
That should already work.

**Reply by smokedlinq** [reply]
I'll double-check, I didn't think it was clearing properties that were null.

**Comment by danielcweber** [comment]
Maybe that's just the overload of Property(x. => x...., null) that does this. Not sure.
